projects/lookup/src/modules/lookup/lookup-show-more-async-modal.component.html

```diff
@@ -0,0 +1,164 @@
+      ]"
+    >
+      <sky-toolbar class="sky-lookup-show-more-modal-toolbar">
+        <sky-toolbar-section>
+          <sky-toolbar-item>
+            <sky-search
+              [debounceTime]="250"
+              [searchText]="searchText"
+              (searchApply)="searchApplied($event)"
+            >
+            </sky-search>
+          </sky-toolbar-item>
+          <sky-toolbar-view-actions>
+            <button
+              *ngIf="context.showAddButton"
+              type="button"
+              class="sky-btn sky-btn-link sky-lookup-show-more-modal-add"
+              (click)="addButtonClicked()"
+            >
+              <sky-icon
+                *skyThemeIf="'modern'"
+                icon="add"
+                iconType="skyux"
+              ></sky-icon>
+              <sky-icon *skyThemeIf="'default'" icon="plus-circle"></sky-icon>
+              {{ 'skyux_lookup_show_more_add' | skyLibResources }}
+            </button>
+          </sky-toolbar-view-actions>
+        </sky-toolbar-section>
+      </sky-toolbar>
+      <sky-toolbar
+        *ngIf="context.selectMode === 'multiple'"
+        class="sky-lookup-show-more-modal-muiltiselect-toolbar"
+      >
+        <sky-toolbar-section>
+          <sky-toolbar-item>
+            <button
+              class="
+                sky-btn sky-btn-link sky-lookup-show-more-modal-select-all-btn
+              "
+              type="button"
+              (click)="selectAll()"
+            >
+              {{
+                'skyux_lookup_show_more_select_all_button_title'
+                  | skyLibResources
+              }}
+            </button>
+          </sky-toolbar-item>
+          <sky-toolbar-item>
+            <button
+              class="
+                sky-btn sky-btn-link sky-lookup-show-more-modal-clear-all-btn
+              "
+              type="button"
+              (click)="clearAll()"
+            >
+              {{
+                'skyux_lookup_show_more_clear_all_button_title'
+                  | skyLibResources
+              }}
+            </button>
+          </sky-toolbar-item>
+          <sky-toolbar-view-actions>
+            <sky-checkbox
+              [(checked)]="onlyShowSelected"
+              (change)="updateDisplayedItems()"
+            >
+              <sky-checkbox-label>
+                {{
+                  'skyux_lookup_show_more_show_selected_option_title'
+                    | skyLibResources
+                }}
+              </sky-checkbox-label>
+            </sky-checkbox>
+          </sky-toolbar-view-actions>
+        </sky-toolbar-section>
+      </sky-toolbar>
+    </div>
+    <div
+      *ngIf="!isSearching && displayedItems.length === 0"
+      class="sky-font-deemphasized sky-lookup-show-more-no-results"
+    >
+      {{ 'skyux_lookup_show_more_no_results' | skyLibResources }}
+    </div>
+    <sky-repeater class="sky-lookup-show-more-repeater" expandMode="none">
```

Comment:
Should we have a `aria-live="assertive"` attribute on the search results to tell screen readers this area is dynamically loading in new content?

Reply:
Let's defer the accessibility changes to a separate set of work, since there are also changes needed to autocomplete to make this component truly accessible.